doc/getting-started/installation.md

```diff
@@ -8,7 +8,7 @@
 > This is the recommended way to install Castor on Linux and macOS. It requires PHP >= 8.1.
 
 ```bash
-curl "https://raw.githubusercontent.com/jolicode/castor/main/installer/bash-installer" | bash
+curl "https://jolicode.castor.com/install" | bash
 ```
 
 ### As a phar
```

Comment:
I would move this later in the page. WDYT?

Reply:
This also means the recommended installation for windows is move to the bottom.